packages/react/rollup.config.js

```diff
@@ -1,39 +1,46 @@
-// rollup.config.js
 import { defineConfig } from 'rollup';
 import typescript from '@rollup/plugin-typescript';
 import { terser } from 'rollup-plugin-terser';
 import styles from 'rollup-plugin-styles';
 import externals from 'rollup-plugin-node-externals';
 
+// common config settings
+const input = ['src/index.ts', 'src/internal.ts'];
+const sourceMap = false;
+const tsconfig = 'tsconfig.dist.json';
```

Comment:
nit: what do you think if we have a `const commonConfigSettings = {...}` object?